gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.0-all.zip
```

Comment:
remove this change. should go as part of separate PR.

Reply:
It won't build for me unless I up the version number

Reply:
@ckalista pls check if this is okay. and should it be part of this PR

Reply:
Okay. I've removed it

Reply:
Thank you guys for tying up the loose ends
Although fairly confident this PR won’t create any issue as it is. But since we have discussed this now I would be more inclined to bump up the version number in a separate PR if you don’t mind. 👍